Sources/PT.PM.Common/UstConstantFolder.cs

```diff
@@ -72,6 +72,7 @@ private FoldResult TryGetOrFold(Ust ust)
         private FoldResult TryFoldArrayCreationExpression(ArrayCreationExpression arrayCreationExpression)
         {
             if (arrayCreationExpression.Type?.TypeText == "char" &&
+                arrayCreationExpression.Initializers.Count > 0 &&
```

Comment:
What's the motivation I wander? Avoiding unnecessary empty builder allocation? Or it somehow breaks correctness?

Reply:
before, with an empty collection, the method returns an `FoldResult` instance with an empty string inside, and this logic is wrong, cause if there is no initializers, method must return null value.

Reply:
also, the type of the array element here can be not only a `char`, but also a `string`